### PR TITLE
MINT-1881 private key based dash

### PR DESCRIPTION
**Pull request by alexander-elgin**

### CODE REVIEW

File: packages/core/services/signers/eth/EthereumTrezorDevice.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import EventEmitter from 'events'
 import EthereumTx from 'ethereumjs-tx'
-import hdkey from 'ethereumjs-wallet/hdkey'
+import hdkey from '../../../../../node_modules/ethereumjs-wallet/hdkey'
```
Comment (ozalexo) [inline packages/core/services/signers/eth/EthereumTrezorDevice.js]: That is why I'm not using autoimport :-)

**Comment by alexander-elgin** [comment]
fixed in 4bfc479337d41460a1d3e80deaf9311eab528544

File: packages/core/services/signers/eth/EthereumTrezorDevice.js (hunk 1 above)
Comment (alexander-elgin) [reply packages/core/services/signers/eth/EthereumTrezorDevice.js]: ашчув шт 4bfc479337d41460a1d3e80deaf9311eab528544

File: packages/login/network/BitcoinEngine.js [hunk 2]
```diff
@@ -145,3 +145,17 @@ export class BCCEngine extends BitcoinEngine {
     }
   }
 }
+
+export class DASHEngine extends BitcoinEngine {
+  _signInputs (txb, inputs, options) {
+    txb.enableBitcoinCash(true)
```
Comment (ozalexo) [inline packages/login/network/BitcoinEngine.js]: I suspect that we need to find out appropriate replacement for the method enableBitcoinCash.

**Comment by alexander-elgin** [comment]
failed to merge the changes with the PR (https://github.com/ChronoBank/ChronoMint/pull/816) applied. See https://github.com/ChronoBank/ChronoMint/pull/826 instead

**Pull request closed without merging**